meta-flutter-apps/conf/flutter-apps.json

```diff
@@ -294,7 +294,7 @@
         "src_files": {
             "example": [
                 "0001-ivi-homescreen-support.patch",
-                "textured_pbr.filamat;destsuffix=git/assets/materials"
+                "textured_pbr.filamat;destsuffix=${BB_GIT_DEFAULT_DESTSUFFIX}/assets/materials"
```

Comment:
this will not get resolved, as the tool scripts aren't Yocto variable aware.

Reply:
It shouldn't be resolved, don't it leave as unresolved bitbake variable?